radixdlt/babylon-wallet-android
# P2P links migration messages

## Description
Because we removed p2pLinks from the profile, the users will lose their p2p links after updating the app and we need to notify the users about this (if they had any links) when opening the app after update. Also, when restoring a profile that contains p2pLinks, since they will be ignored (and erased from the profile JSON), we need to notify the users about this as well.


## How to test

App update use-case:

1. Have a 1.5.4 app build with at least 1 linked connector.
2. Install the new app version
3. Check that the message about relinking connectors is displayed

Profile restore use-case:
1. Restore a profile containing legacy p2pLinks
2. Complete the onboarding
3. Check that the message about relinking connectors is displayed

## Screenshot

![share_8845757547680704248](https://github.com/radixdlt/babylon-wallet-android/assets/164897324/559846d4-07f8-47a5-ac30-532d5ba9537e)


## PR submission checklist
- [ ] I have tested "Re-link connectors" message after app update
- [ ] I have tested "Re-link connectors" message after profile restore
